projects/osx_rules

```diff
@@ -2,5 +2,5 @@
 
 #---------------------------------------------------------------------------------
 %.app: $(OFILES)
-	$(CXX) $(LDFLAGS) -arch arm64 -framework Cocoa -framework Carbon -o $@ $(OFILES) $(LIBS) -L/opt/homebrew/Cellar/sdl12-compat/1.2.60/lib
+	$(CXX) $(LDFLAGS) -arch $(shell uname -m) -framework Cocoa -framework Carbon -o $@ $(OFILES) $(LIBS) -L$(shell brew --prefix sdl12-compat)/lib
```

Comment:
thanks for fixing this 👍🏻  I ran into the same issue on macos just yest as I installed sdl12-compat from homebrew and the latest current version is `1.2.68` so this should fix it going forward.